README.md

```diff
@@ -39,13 +39,42 @@ The meeting schedule, issue, etc can be configured with inouts to this action.
 - `meetingLabels`: The labels to apply to issues generated by this GitHub Action for meetings. Default: `meeting`
 - `agendaLabel`: Label to pull the agenda from. Default: `meeting-agenda`
 - `issueTitle`: Template string for issue title.  Default: `Meeting <%= date.toFormat('yyyy-MM-dd') %>`
-- `issueTemplate`: The name of the issue template found in `.github/ISSUE_TEMPLATE`. Default: `meeting.md`
+- `issueTemplate`: The name of the issue template found in `.github/ISSUE_TEMPLATE`. Default: `meeting.md` 
 
 ### Default Issue Template
 
-The default issue template can be seen here: https://github.com/wesleytodd/meeting-maker/issues/34
+The default issue template can be seen in the ISSUE_TEMPLATES directory, as [meeting.md](https://github.com/pkgjs/meet/blob/main/.github/ISSUE_TEMPLATE/meeting.md). Additionally, you can find example issues of the latest version via the closed issues that are the result of CI tests.
 
-It is based off the one commonly used on across the Node.js Org, but any additions or improvements are welcome.
+Our default template is based off of the practices of the Node.js project's practices around meetings, but changes and enhancements are welcome.
```

Comment:
Is it correct to have "practices" twice in this sentence?

Reply:
```suggestion
Our default template is based off of the Node.js project's practices around meetings, but changes and enhancements are welcome.

```